go/tasks/plugins/webapi/agent/plugin.go

```diff
@@ -162,6 +163,8 @@ func (p Plugin) Status(ctx context.Context, taskCtx webapi.StatusContext) (phase
 	taskInfo := &core.TaskInfo{}
 
 	switch resource.State {
+	case admin.State_PENDING:
+		return core.PhaseInfoQueued(time.Now(), core.DefaultPhaseVersion, "Job Created"), nil
```

Comment:
sorry, I think we should use PhaseInfoQueuedWithTaskInfo here. https://github.com/flyteorg/flyteplugins/blob/552f145c4632944a23e7e2d9fa2b2f065c4b14b9/go/tasks/plugins/webapi/bigquery/plugin.go#L287

Reply:
No problem!
Thanks a lot for spending time reviewing this!